CHANGELOG.md

```diff
@@ -6,36 +6,58 @@ Swift 4.2
 
 * [SE-209](https://github.com/apple/swift-evolution/blob/master/proposals/0209-package-manager-swift-lang-version-update.md)
 
-    The `swiftLanguageVersions` property no long takes its Swift language versions via
-    a freeform Integer array; instead it should be passed as a new `SwiftVersion` enum
-    array.
+  The `swiftLanguageVersions` property no long takes its Swift language versions via
+  a freeform Integer array; instead it should be passed as a new `SwiftVersion` enum
+  array.
 
 * [SE-208](https://github.com/apple/swift-evolution/blob/master/proposals/0208-package-manager-system-library-targets.md)
 
-    The `Package` manifest now accepts a new type of target, `systemLibrary`. This
-    deprecates "system-module packages" which are now to be included in the packages
-    that require system-installed dependencies.
+  The `Package` manifest now accepts a new type of target, `systemLibrary`. This
+  deprecates "system-module packages" which are now to be included in the packages
+  that require system-installed dependencies.
 
 * [SE-201](https://github.com/apple/swift-evolution/blob/master/proposals/0201-package-manager-local-dependencies.md)
 
-    Packages can now specify a dependency as `package(path: String)` to point to a
-    path on the local filesystem which hosts a package. This will enable interconnected
-    projects to be edited in parallel.
+  Packages can now specify a dependency as `package(path: String)` to point to a
+  path on the local filesystem which hosts a package. This will enable interconnected
+  projects to be edited in parallel.
 
 * The `generate-xcodeproj` has a new `--watch` option to automatically regenerate the
-    Xcode project if changes are detected. This uses the
-    [`watchman`](https://facebook.github.io/watchman/docs/install.html) tool to detect
-    filesystem changes.
+  Xcode project if changes are detected. This uses the
+  [`watchman`](https://facebook.github.io/watchman/docs/install.html) tool to detect
+  filesystem changes.
 
 * Scheme generation has been improved:
   * One scheme containing all regular and test targets of the root package.
   * One scheme per executable target containing the test targets whose dependencies
     intersect with the dependencies of the exectuable target.
 
+* Packages which mix versions of the form `vX.X.X` with `Y.Y.Y` will now be parsed and
+  ordered numerically.
+
+* Local `Package` declarations can no longer specify an absolute path, and must instead
+  be relative definitions.
+
+* `swift test` now has rudimentary support for exporting xUnit-formatted xml output.
+
+* The package manager now supports `ppc64le`, `arm`, `arm64`, and `aarch64` machines.
```

Comment:
This is not really true, we should keep it out of the change log

Reply:
I was looking at #1482, #1546, #1587— I'll delete this line 👍